environments/__init__.py

```diff
@@ -1 +1,2 @@
-from .base import EnvironmentBase
\ No newline at end of file
+from .base import EnvironmentBase
+# from .multi_armed_bandit import MultiArmedBanditEnvironment
```

Comment:
How should we import it without having a loop in the imports? Is there any method for that?